e2e/client/playwright/custom-filed.spec.ts

```diff
@@ -0,0 +1,120 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {Settings} from './page-object-models/settings';
+import {restoreDatabaseSnapshot, s} from './utils';
+
```

Comment:
there's a spelling mistake - should be `field` not `filed`. Also, since there are multiple possible custom fields, not only one, it should `fields`. Also let's add a prefix so make it `authoring.custom-fields.spec.ts`